common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -219,7 +221,7 @@ def get_html(self, view=STUDENT_VIEW):
 
         track_status = (self.download_track and self.track)
         transcript_download_format = self.transcript_download_format if not track_status else None
-        sources = filter(None, self.html5_sources)
+        sources = [_f for _f in self.html5_sources if _f]
```

Comment:
Same issue with the placeholder here as above; maybe replace `_f` with something like `source` here?